src/Sanitizer.php

```diff
@@ -16,8 +16,14 @@ class Sanitizer
      */
     private array $filters;
 
-    public function __construct()
+    private array $removeBlocks = ['script', 'iframe', 'object'];
+    
+    private array $removeAttributtes = [];
+
+    public function __construct(array $removeBlocks = [], array $removeAttributtes = [])
```

Comment:
I added type declarations here, but online editor added too many spaces :doh:. Dev dependency requirements have ECS included which you can run locally to fix this and the missing declaration in `RemoveAttributes`.

```sh
./vendor/bin/ecs --fix
```